src/Roact.lua

```diff
@@ -9,7 +9,8 @@ if not roactModule then
 		local TS = require(game:GetService("ReplicatedStorage"):WaitForChild("rbxts_include"):WaitForChild("RuntimeLib"))
 		Roact = TS.import(script, TS.getModule(script, "roact").src)
 	else
```

Comment:
Assuming this is a *roblox-ts plugin* (not a very common use!) when Roact can't otherwise be found seems a bit obtuse.